### ISSUE TITLE
There's only one capital letter D in Datadog :)

### ISSUE THREAD

**Issue by phrawzty** [containous/traefik#5268]
### Do you want to request a *feature* or report a *bug*?

Bug

### Did you try using a 1.7.x configuration for the version 2.0?

- [ ] Yes
- [x] No

### What did you do?

I noticed that "Datadog" was stylised incorrectly in the docs (examples: [here](https://docs.traefik.io/configuration/tracing) and [here](https://docs.traefik.io/master/observability/tracing/datadog)). So, I grepped across the repo for every instance of the `DataDog` and `Datadog` strings, respectively.

It's not that big a deal, but the "dog" in Datadog—as a trademarked term—is written with a lower-case letter "d". I can totally understand not wanting to alter it in the code, but it'd be nice to have it written correctly in the documentation. Again, not a big deal—just letting y'all know. 😄

Cheers!

### What did you expect to see?

```
repos/traefik (master) » grep -r DataDog * | wc -l
      0
repos/traefik (master) » grep -r Datadog * | wc -l
      75
```

### What did you see instead?

```
repos/traefik (master) » grep -r DataDog * | wc -l
      61
repos/traefik (master) » grep -r Datadog * | wc -l
      14
```

### Output of `traefik version`: (_What version of Traefik are you using?_)

n/a

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

n/a

### If applicable, please paste the log output in DEBUG level (`--log.level=DEBUG` switch)

n/a

**Comment by dduportal** [comment]
Hi @phrawzty , thanks for the notice! Would you be willing to contribute such a change in the documentation ?

**Comment by ldez** [comment]
The Datadog organization use `DataDog` https://github.com/DataDog/ some all the imports use `DataDog`, it's not possible to obtain:
```
repos/traefik (master) » grep -r DataDog * | wc -l
      0
```

**Comment by phrawzty** [comment]
@dduportal Sure, I'll prepare a PR. 👍 

@ldez Correct. That's why I specifically said :
> I can totally understand not wanting to alter it in the code, but it'd be nice to have it written correctly in the documentation. 

The `grep` output was what I _expected_ (as per the Issue Template)—not what is possible. 🙂

**Comment by ldez** [comment]
I already fixed the problem in #5272 :wink:

**Comment by traefiker** [comment]
Closed by #5272.
